packages/kcmsf/src/pages/matchResult.tsx

```diff
@@ -0,0 +1,147 @@
+          tema1GoalStateData.length === 0
+            ? null
+            : Math.min(
+                ...tema1GoalStateData.map((result) => result.goalTimeSeconds)
+              );
+        const team2GoalTime =
+          tema2GoalStateData.length === 0
+            ? null
+            : Math.min(
+                ...tema2GoalStateData.map((result) => result.goalTimeSeconds)
+              );
+        //　結果を作成
+        const team1Result = {
+          teamID: team1ID,
+          points: team1Point,
+          goalTimeSeconds: team1GoalTime,
+        } as RunResult;
+        const team2Result = {
+          teamID: team1ID,
+          points: team2Point,
+          goalTimeSeconds: team2GoalTime,
+        } as RunResult;
+
+        setRightTeamResult(team1Result);
+        setLeftTeamResult(team2Result);
+      }
+    }
+  }, [match, matchInfo]);
+  return (
+    <Flex
+      h="100%"
+      direction="column"
+      gap="md"
+      align="center"
+      justify="center"
+      mx="2rem"
+    >
+      {matchInfo && (
+        <Flex align="center" direction="column" justify="center" gap="sm">
+          <Title order={1}>{config.contestName}</Title>
+          <Title order={2}>
+            {config.match[matchInfo?.matchType].name} #{match?.matchCode}{" "}
+            試合結果
+          </Title>
+        </Flex>
+      )}
+      <Flex align="center" justify="center">
+        <Flex pb="sm" gap="lg">
+          <Text size="2.5rem" c="blue">
+            {matchInfo?.teams.left?.teamName}
+          </Text>
+          <Text size="2.5rem">vs</Text>
+          <Text size="2.5rem" c="red">
+            {matchInfo?.teams.right?.teamName}
+          </Text>
+        </Flex>
+      </Flex>
+      <Text size="2.5rem">得点</Text>
+      <Flex align="center" justify="center">
+        <Flex pb="sm" gap="lg">
+          <Text size="3rem" c="blue">
+            {leftTeamResult ? leftTeamResult.points + "点" : "結果無し"}
+          </Text>
+          <Text size="3rem">-</Text>
+          <Text size="3rem" c="red">
+            {rightTeamResult ? rightTeamResult.points + "点" : "結果無し"}
+          </Text>
+        </Flex>
+      </Flex>
+      <Text size="2.5rem">ゴールタイム</Text>
+      <Flex align="center" justify="center">
+        <Flex pb="sm" gap="lg">
+          <Text size="3rem" c="blue">
+            {leftTeamResult && leftTeamResult?.goalTimeSeconds !== null
+              ? parseSeconds(leftTeamResult.goalTimeSeconds)
+              : "フィニッシュ"}
+          </Text>
+          <Text size="3rem">-</Text>
+          <Text size="3rem" c="red">
+            {rightTeamResult && rightTeamResult?.goalTimeSeconds !== null
+              ? parseSeconds(rightTeamResult.goalTimeSeconds)
+              : "フィニッシュ"}
+          </Text>
+        </Flex>
+      </Flex>
+      <Flex mt="1rem">
+        <Button mx="3rem" flex={1}>
+          <Link to={"/matchlist"} style={{ color: "white" }}>
+            <Text>試合表に戻る</Text>
+          </Link>
+        </Button>
```

Comment:
```suggestion
        <Button
          component={Link}
          to="/matchlist"
          leftSection={<IconArrowLeft />}
          variant="outline"
        >
          試合表に戻る
        </Button>
```
- 不要なスタイルが付いています。
- Polymorphic Componentを使ってください。
- 他の箇所では`filled`なボタンを"決定・進む"の意味で用いていると思うので、`variant`は`outline`か`subtle`がよいでしょう。
- 何かアイコンを付けてください。上はただの例ですが、"戻る"を意味するようなアイコンが好ましいです。